Products/CMFPlone/browser/jsvariables.py

```diff
@@ -36,6 +37,7 @@ def __call__(self, *args, **kwargs):
 
         props = getToolByName(context, 'portal_properties').site_properties
         portal_url = getToolByName(context, 'portal_url')()
+        base_url = self.request['HTTP_REFERER']
```

Comment:
no, the referrer identifies the URI that linked to the resource being requested; that is, where the request was originated. that's not the `base_url`, AFAIK.